tests/System/client_core.sh

```diff
@@ -4,106 +4,156 @@
 #
 # Submitter should follow through the logs
 
-echo " "
-echo " "
+echo
+echo
 echo " ########################## REAL BASICS #############################"
-echo " "
-echo " "
+echo
+echo
 
-echo "dirac-proxy-init"
-dirac-proxy-init
+echo "========================"
+echo "===  dirac-proxy-init -U"
+echo
+dirac-proxy-init -C $SERVERINSTALLDIR/user/client.pem -K $SERVERINSTALLDIR/user/client.key -U
```

Comment:
This is a system test, and normally $SERVERINSTALLDIR is not defined, as normally it's run within the certification process, defined in this trello card: https://trello.com/c/1IPtPhds (the environment there is simply what results in a DIRAC client installation, so created with dirac-install and nothing else).

If you want to run this in Jenkins, then you should use not $SERVERINSTALLDIR but $TESTCODE.

Reply:
Test has been changed to run on Jenkins and if variables will not defined